symfony/client/src/orangehrmAdminPlugin/pages/workShift/SaveWorkShift.vue

```diff
@@ -0,0 +1,203 @@
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6" class="orangehrm-main-title">Add Work Shift</oxd-text>
+
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                :label="$t('admin.shift_name')"
+                :rules="rules.name"
+                v-model="workShift.name"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+
+        <oxd-form-row>
+          <oxd-text class="orangehrm-sub-title">Working Hours*</oxd-text>
+          <oxd-grid :cols="4" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                type="time"
+                :rules="rules.fromTime"
+                :label="$t('general.from')"
+                v-model="workShift.startTime"
+              />
+            </oxd-grid-item>
+
+            <oxd-grid-item>
+              <oxd-input-field
+                type="time"
+                :rules="rules.endTime"
+                :label="$t('general.to')"
+                v-model="workShift.endTime"
+              />
+            </oxd-grid-item>
+
+            <oxd-grid-item>
+              <oxd-input-group :label="$t('admin.duration_per_day')">
+                <oxd-text class="orangehrm-workshift-duration" tag="p">
+                  {{ selectedTimeDuration }}
+                </oxd-text>
+              </oxd-input-group>
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <work-shift-employee-autocomplete
+                v-model="workShift.empNumbers"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button displayType="ghost" label="Cancel" @click="onCancel" />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {navigate} from '@orangehrm/core/util/helper/navigation';
+import {APIService} from '@orangehrm/core/util/services/api.service';
+import {
+  endTimeShouldBeAfterStartTime,
+  required,
+  shouldNotExceedCharLength,
+  validTimeFormat,
+} from '@orangehrm/core/util/validation/rules';
+import {diffInTime, secondsTohhmm} from '@/core/util/helper/datefns';
+import WorkShiftEmployeeAutocomplete from '@/orangehrmAdminPlugin/components/WorkShiftEmployeeAutocomplete';
+
+const workShiftModel = {
+  id: '',
+  name: '',
+  hoursPerDay: '',
+  startTime: '08:00',
```

Comment:
we may have to get default `startTime`, `endTime` as props, since those are config values in BE

Reply:
done